incubator/kube-aws-ingress-controller/templates/daemonset.yaml

```diff
@@ -0,0 +1,55 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: {{ template "kube-aws-ingress-controller.name" . }}-skipper-ingress
+  namespace: kube-system
+  labels:
+    component: ingress
+spec:
+  selector:
+    matchLabels:
+      component: ingress
+  updateStrategy:
+    type: RollingUpdate
+  template:
+    metadata:
+      name: {{ template "kube-aws-ingress-controller.name" . }}-skipper-ingress
+      labels:
+        component: ingress
+    spec:
+      {{- if .Values.rbac.create }}
```

Comment:
Remove the if block. Use a named template as shown here for references to the service account:

https://github.com/kubernetes/helm/blob/master/docs/chart_best_practices/rbac.md